src/plugins/workspace/public/components/workspace_overview/types.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AppCategory } from '../../../../../core/types';
+
+export interface GetStartCard {
```

Comment:
appId, category seems duplicate to what plugins have register for `App`, do we consider expanding `application.register` on 2.15 to add such card?

Reply:
changed to `export interface GetStartCard extends Partial<App>` to reuse `App`, so plugin only need add three more fields `featureName`, `featureDescription` and `link`